cylc/flow/scheduler.py

```diff
@@ -626,7 +625,7 @@ async def start_scheduler(self):
         finally:
             self.profiler.stop()
 
-    async def run(self, daemonize=False):
```

Comment:
Probably removed some time ago. We still have the `daemonize` function elsewhere. This `daemonize` flag is never used here. Also doesn't seem to be used by the callers of this function.